### DISCUSSION TITLE
Too many requests error for transfers

### DISCUSSION THREAD

**Discussion by Kelos621** [iterate-ch/cyberduck#15890] [Q&A]
![Screenshot_1](https://github.com/iterate-ch/cyberduck/assets/168132606/1330845d-2abd-4872-8f21-4d70a545f4a4)

**Comment by dkocher** [comment]
There are two things to mitigate this problem

- Enable _Repeat failed networking tasks_ in [Preferences → Connection](https://docs.cyberduck.io/cyberduck/connection/#repeat-failed-networking-tasks).
- Lower the [number of connections allowed](https://docs.cyberduck.io/cyberduck/transfer/#limit-number-of-parallel-connections) in transfers.
